include/nbl/builtin/hlsl/spirv_intrinsics/glsl.std.450.hlsl

```diff
@@ -0,0 +1,46 @@
+#ifndef _NBL_BUILTIN_HLSL_SPIRV_INTRINSICS_GLSL_STD_450_INCLUDED_
+#define _NBL_BUILTIN_HLSL_SPIRV_INTRINSICS_GLSL_STD_450_INCLUDED_
+
+#ifdef __HLSL_VERSION
+#include <nbl/builtin/hlsl/cpp_compat.hlsl>
+#include "spirv/unified1/GLSL.std.450.h"
+
+namespace nbl 
+{
+namespace hlsl
+{
+namespace spirv
+{
+
+// Find MSB and LSB restricted to 32-bit width component types https://registry.khronos.org/SPIR-V/specs/unified1/GLSL.std.450.html
+
+template<typename Integral>
+[[vk::ext_instruction(GLSLstd450::GLSLstd450FindILsb, "GLSL.std.450")]]
+enable_if_t<is_integral_v<Integral> && (sizeof(scalar_type_t<Integral>) == 4), Integral> findILsb(Integral value);
+
+template<typename Integral>
+[[vk::ext_instruction(GLSLstd450::GLSLstd450FindSMsb, "GLSL.std.450")]]
+enable_if_t<is_integral_v<Integral> && (sizeof(scalar_type_t<Integral>) == 4), Integral> findSMsb(Integral value);
+
+template<typename Integral>
+[[vk::ext_instruction(GLSLstd450::GLSLstd450FindUMsb, "GLSL.std.450")]]
+enable_if_t<is_integral_v<Integral> && (sizeof(scalar_type_t<Integral>) == 4), Integral> findUMsb(Integral value);
```

Comment:
I think these you can just not template because `Integral` can only be `uint32_t` or `int32_t` for the who functions

Reply:
hence `(sizeof(scalar_type_t<Integral>) == 4)`

Reply:
I mean that `S` in the intrinsic means `int` (signed) only and `U` means `uint` (unsigned) only so there really isn't any room for any other type

Reply:
makes sense